docs/using-hive.md

```diff
@@ -311,6 +325,29 @@ platform:
     vCenter: vcenter.example.com
 ```
 
+For Openstack, replace the contents of `compute.platform` with:
+```yaml
+  openstack:
+    type: m1.large
+```
+Note: Use an instance type that meets the minimum requirement for the version of OpenShift being installed.
+
+and replace the contents of `controlPlane.platform` with:
+```yaml
+  openstack:
+    type: ci.m4.xlarge
+```
+Note: Use an instance type that meets the minimum requirement for the version of OpenShift being installed.
+
+and replace the contents of `platform` with:
+```yaml
+  openstack:
+    cloud: mycloud
+    computeFlavor: m1.large
+    externalNetwork: provider_net_shared_3
```

Comment:
can we turn this into a more generic-looking value? something like "openstack_network_name"

Reply:
done